src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -362,6 +362,11 @@ public static FormulaValue Text(FormattingInfo formatInfo, IRContext irContext,
 
             if (formatString != null && !TextFormatUtils.IsValidFormatArg(formatString, formatInfo.CultureInfo, out textFormatArgs))
             {
+                if (formatString.StartsWith("[$-", StringComparison.OrdinalIgnoreCase) && !(textFormatArgs.HasDateTimeFmt && textFormatArgs.HasNumericFmt))

```

Comment:
>formatString.StartsWith("[$-", StringComparison.OrdinalIgnoreCase) [](http://example.com/codeflow?start=20&length=66)

There is a similar condition at `TextFormatUtils.cs` file (line 147) but with a different `StringComparison`. Is there a reason why? Just curious.

Reply:
want to know what failed for bad language code. other cases will return invalid arg error.